docs/complex-transactions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Send Complex Transaction with the Flow CLI
+sidebar_title: Build a Complex Transaction
+description: How to build and send a complex Flow transaction from the command line
+---
+
+The Flow CLI provides commands to build, sign and send transactions allowing you to specify different authorizers, signers and proposers.  
+
+The process of sending a complex transactions includes three steps:
+1. [build a transaction](/build-transactions/)
+2. [sign the built transaction](/sign-transaction/)
+3. [send signed transaction](/send-signed-transactions/)
+
+Read more about each command flags and arguments in the above links.
+
+## Examples
+We will describe common examples for complex transactions. All examples are using an example configuration [defined here]().
+
+### Single payer, proposer and authorizer
+The simplest Flow transaction declares a single account as the proposer, payer and authorizer.
+
+Build the transaction:
+```shell
+> flow transactions build tx.cdc 
+  --proposer alice 
+  --payer alice 
+  --authorizer alice 
+  --filter payload --save tx1
+```
+Sign the transaction:
+```shell
+> flow transactions sign tx1 --signer alice 
+  --filter payload --save tx2
+```
+Submit the signed transaction:
+```shell
+> flow transactions send-signed tx2
+```
+Transaction content (`tx.cdc`):
+```
+transaction {
+    preapre(signer: AuthAccount) {}
+    execute { ... }
+}
+```
+
+### Single payer and proposer, multiple authorizers
+A transaction that declares same payer and proposer but multiple authorizers each required to sign the transaction. Please note that the order of signing is important, and [the payer must sign last](https://docs.onflow.org/concepts/transaction-signing/#payer-signs-last).
+
+Build the transaction:
+```shell
+> flow transactions build tx.cdc 
+  --proposer alice
+  --payer alice
+  --authorizer bob
+  --authorizer charlie 
+  --filter payload --save tx1
+```
+Sign the transaction with authorizers:
+```shell
+> flow transactions sign tx1 --signer bob
+  --filter payload --save tx2
+```
+```shell
+> flow transactions sign tx2 --signer charlie
+  --filter payload --save tx3
+```
+Sign the transaction with payer:
+```shell
+> flow transactions sign tx3 --signer alice
+  --filter payload --save tx4
+```
+Submit the signed transaction:
+```shell
+> flow transactions send-signed tx4
+```
+Transaction content (`tx.cdc`):
+```
+transaction {
+    preapre(signer1: AuthAccount, signer2: AuthAccount) {}
```

Comment:
Perhaps it would be nice to be a bit more descriptive here, because there are different participants.
```suggestion
    preapre(bob: AuthAccount, charlie: AuthAccount) {}
```